examples/drum-circle.pl

```diff
@@ -5,9 +5,13 @@
 # Adapted from 'drum-circle' by Gene Boggs
 # https://github.com/ology/Music
 
+use if $ENV{USER} eq 'gene', lib => map { "$ENV{HOME}/sandbox/$_/lib" } qw(MIDI-Drummer-Tiny MIDI-Util Music-Duration-Partition);
```

Comment:
I am not sure why these lines are showing up in the PR. They are *not* in the committed and pushed file. WTF :\

Reply:
It's not called "git" because it's nice to use and friendly :)